akka/src/main/scala/chatapp/Chat.scala

```diff
@@ -16,81 +16,79 @@ object Action extends Enumeration {
 import Action._
 
 class BehaviorFactory(compute: Int, post: Int, leave: Int, invite: Int) {
-  def apply(dice: DiceRoll): Action = {
-    val pick = dice()
+  val _compute: Int = compute
+  val _post: Int = _compute + post
+  val _leave: Int = _post + leave
+  val _invite: Int = _leave + invite
+
+  def apply(pick: Int): Action = {
```

Comment:
Why this change? Pony passes also the `DiceRoll` object to the apply method: https://github.com/sblessing/chat-app/blob/master/pony/chat.pony#L47

Reply:
In CAF we changed it that way to have only one random generator, pony use only one too.

Reply:
Ehm, here we only got one random number generator, too.
`DiceRoll` is simply a decorator for the PRNG. As it is in Pony.
To stay in sync with Pony, I think this change is also better reverted.

Reply:
I used CAF as reference for the rework, I thought that was changed to pony too.

Reply:
I think we refer to Pony as the reference implementation, and please see my link above, it's still there, no?

Reply:
ok, I will change it back.

Reply:
done